libdrgn/stack_trace.c

```diff
@@ -109,6 +109,7 @@ drgn_stack_trace_num_frames(struct drgn_stack_trace *trace)
 LIBDRGN_PUBLIC struct drgn_error *
 drgn_format_stack_trace(struct drgn_stack_trace *trace, char **ret)
 {
+	struct drgn_error *err = NULL;
```

Comment:
No need to initialize this. Let's allow the compiler to warn us if we forget to set this.
```suggestion
	struct drgn_error *err;
```